docs/modules/preprocessing.rst

```diff
@@ -30,6 +30,7 @@ transfer between two transport modes). We can cluster these into locations that
 frequently visits.
 
 .. autofunction:: trackintel.preprocessing.staypoints.generate_locations
+.. autofunction:: trackintel.preprocessing.staypoints.create_activity_flag
```

Comment:
should we include some description of this function? for example just simply adding:

> Staypoints are points where someone stayed for a longer period of time (e.g., during a transfer between two transport modes). We can cluster these into locations that a user frequently visits **and/or infer if they correspond to activities.**

Reply:
Yes! We should show them what we have ;-) 
Is this in the readme?

Reply:
This is just 4 lines above :).